### DISCUSSION TITLE
Feedback About Color-Blindness Themes Belongs In This Forum

### DISCUSSION THREAD

**Discussion by adlai** [community/community#48608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![freetime](https://user-images.githubusercontent.com/94171/221385002-7ae02a9c-f195-447b-9c76-bf879324ac60.png)

As usual, I'm complaining about the complaint platform, instead of any specific feature about which complaints have been invited. In this case, I clicked the Give feedback link near the bottom of the modal in the above screenshot, and apparently nothing happened; moments later, I noticed that while the address bar had been rewritten, and the page behind the modal had navigated, it simply went to [General](https://github.com/community/community/discussions) instead of [Accessibility](https://github.com/community/community/discussions/categories/accessibility).

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @adlai! We're happy you're here. After reviewing your issue, it seems that the Accessibility category might not be the most applicable as your issue is about the unexpected behavior when using the feedback system and not specific to making the product more accessible. I've asked @queenofcorgis to review this issue and move it into a more accurate category for you, if applicable. Thank you!
